invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/frontpage.less

```diff
@@ -72,26 +73,189 @@ only screen and (max-width: 993px) {
     flex-direction: row;
 }
 
-.goto-button {
-    text-align: center;
-    padding: 20px 10px;
-    white-space: nowrap;
-}
-
 .goto-button a {
     text-decoration: none;
-    border-radius: 4px;
-    line-height: 25px;
-    padding: 15px 30px;
-    background-color: #0377cd;
-    color:  white !important;
-    height: 50px;
+    color:  white;
     font-size: 1em;
-    font-weight: 200;
+    font-weight: 600;
 }
 
 .goto-button a:hover,
 .goto-button a:active {
-    color: white !important;
-    background-color: #01599c;
+    color: white;
+}
+
+.section-content-dark-bg,
+.section-content-white-bg,
+.section-content-light-bg {
+  padding-top: calc(30px);
+  padding-bottom: calc(2 * 30px);
+
+  &:first-of-type {
+    padding-top: 10px;
+  }
+}
+
+.section-content-dark-bg {
+  padding-top: calc(30px) !important;
+  padding-bottom: calc(2 * 30px);
+}
+
+.section-content-white-bg {
+  padding-top: calc(40px) !important;
+  padding-bottom: calc(2 * 30px);
+}
+
+.section-content-light-bg {
+  padding-top: calc(60px) !important;
+  padding-bottom: calc(2 * 30px);
+
+  &:last-of-type {
+    // NOTE: Fixes spacing between content and footer in the frontpage
+    // TODO: Fix body content padding/margins in general
+    margin-bottom: -1rem;
+  }
+}
+
+.command-line a,
+.hp-blog-section a {
+  text-decoration: none;
+  color: #0267b2;
+  font-weight: 600;
+}
+
+.command-line a:hover,
+.command-line a:focus,
+.hp-blog-section a:hover,
+.hp-blog-section a:focus {
+  color: #0377cd;
+}
+
+.section-content-white-bg,
+.section-content-light-bg {
+  color: #333;
+  width: 100%;
+  text-align: justify;
+  font-size: 1.2em;
+  font-weight: 300;
+}
+
+.section-content-light-bg {
+  background-color: rgba(173, 206, 224, 0.52);
+}
+
+.section-content-white-bg {
+  background-color: #fff;
```

Comment:
I guess you can use `@white` here, no?

Reply:
sure but I prefer to leave it aside and refactor colors separately, otherwise it will be super difficult to keep track of all the changes. I plan to introduce it gradually.